packages/react-components/src/molecules/ExternalLink.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import css from '@emotion/css';
+import { Link } from '../atoms';
+import { externalLinkIcon } from '../icons';
+import { fern, pine } from '../colors';
+import { perRem } from '../pixels';
+
+const containerStyles = css({
+  display: 'flex',
+});
+const styles = css({
```

Comment:
Can you organise and group these properties a bit? E.g. if `padding` isn't `12 - borderWidth`, nor located close to the border definition, you're not going to remember to adjust the padding if the border width is changed